src/components/views/spaces/SpacePanel.tsx

```diff
@@ -399,7 +409,23 @@ const SpacePanel: React.FC = () => {
                             )}
                         </Droppable>
 
-                        <QuickSettingsButton isPanelCollapsed={isPanelCollapsed} />
+                        {SettingsStore.getValue("feature_video_rooms") && (
```

Comment:
What do video rooms have to do with anything here?

Reply:
Oversight on my part